include/glow/Base/Type.h

```diff
@@ -113,6 +113,31 @@ struct ShapeNCHW {
   }
 };
 
+struct ShapeTLBR {
```

Comment:
Is it possible to store ShapeTLBR inside a Node and Instr? It would be more cleaner than converting it to generic array back and forth.

Reply:
I don't think it is easy to have ShapeTLBR in NodeGen since several functions other llvm types have should be implemented as well (e.g. overload operator "==", type conversion)